src/test/java/org/isf/ward/Tests.java

```diff
@@ -592,20 +592,20 @@ public void testMgrValidationCodeAlreadyExists() throws Exception {
 	}
 
 	@Test
-	public void testWardConstructor() throws Exception {
+	void testWardConstructor() throws Exception {
 		Ward ward = new Ward("Z", null, null, null, "TestEmail@gmail.com", 1, 1, 1, false, true);
 		assertThat(ward.getLock()).isNull();
 		ward.setLock(-1);
 		assertThat(ward.getLock()).isEqualTo(-1);
 	}
 
 	@Test
-	public void testWardEquals() throws Exception {
+	void testWardEquals() throws Exception {
 		Ward ward = testWard.setup(false);
-		assertThat(ward).isEqualTo(ward);
 		assertThat(ward)
-				.isNotNull()
-				.isNotEqualTo("someString");
+			.isEqualTo(ward)
```

Comment:
here too

Reply:
Removed the test to itself as it is done later in the method.